frontend/public/components/utils/dropdown.jsx

```diff
@@ -435,7 +436,7 @@ export class ContainerDropdown extends React.PureComponent {
     const dropdownItems = _.mapValues(_.merge(containers, initContainers), containerLabel);
     const title = _.get(dropdownItems, currentKey) || containerLabel(firstContainer);
     return <Dropdown
-      className="btn-group"
+      wrapperClassName="btn-group"
```

Comment:
`wrapperClassName` is a bit confusing. It's not clear what the "wrapper" element really is without looking at the dropdown source.

I'd probably just keep `className` for the dropdown itself and `buttonClassName` for the button. (The important thing is to have a separate prop for the child button class.)